src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -448,6 +491,14 @@ private static void ConfigureAuthentication(WebApplicationBuilder builder, Dashb
                             .RequireAuthenticatedUser()
                             .Build());
                     break;
+                case FrontendAuthMode.BrowserToken:
+                    options.AddPolicy(
+                        name: FrontendAuthorizationDefaults.PolicyName,
+                        policy: new AuthorizationPolicyBuilder(
+                            FrontendAuthenticationDefaults.AuthenticationScheme)
+                            .RequireAuthenticatedUser()
```

Comment:
Is an auth cookie the only way for this condition to be fulfilled? It's an ironclad guarantee that no other auth mechanism (NTLM, client cert, whatever) can fulfill this condition?

Reply:
Hmm, I'm not 100% sure. I think the way to be sure would be to have a specific claim added by cookies auth when mode is BrowserToken and then requiring it here.

What do you think?

Reply:
I've updated the authz policy to require a specific claim that is only set when cookie auth is signed in via a browser token.